crates/pevm/tests/mainnet.rs

```diff
@@ -9,9 +9,11 @@ use pevm::chain::{PevmChain, PevmEthereum};
 pub mod common;
 
 // TODO: [tokio::test]?
-#[test]
+#[tokio::test]
```

Comment:
Unlike `tokio::main`, the default flavour for `tokio::test` is actually single-threaded! We must add the `multi_thread` flavour here, otherwise it would:
- Panic with `tokio::task::block_in_place`.
- Deadlock when the provided runtime is used alongside the global one (like using `await` on `provider.get_block`) with `std::thread::scope`.